template/sbt.go

```diff
@@ -13,7 +13,7 @@ sbtscratch:
     {{.ScratchVolumes}}
 {{end}}
 sbt: &sbt
-  image: arch-docker.eng.lancope.local:5000/sbt
+  image: hseeberger/scala-sbt
```

Comment:
Embedding a non-official image into our templates makes me nervous. We absolutely need to replace the arch-docker image with _something_, but I'm not sure what it is.